rails-snow-queen/app/assets/javascripts/quotingPanel.js

```diff
@@ -14,16 +14,21 @@ class QuotingPanel {
   showAddress(geocodedAddress) {
     if (geocodedAddress) {
       this.geocodedAddress = geocodedAddress;
-      const addressNode = document.getElementById("displayAddress");
+      const primaryAddressNode = document.getElementById("primaryAddress");
+      const secondaryAddressNode = document.getElementById("secondaryAddress");
       let splitAddress = geocodedAddress.split(",");
-      while (addressNode.firstChild) { addressNode.removeChild(addressNode.firstChild) }
-      splitAddress.forEach(element => {
+      while (primaryAddressNode.firstChild) { primaryAddressNode.removeChild(primaryAddressNode.firstChild) }
+      while (secondaryAddressNode.firstChild) { secondaryAddressNode.removeChild(secondaryAddressNode.firstChild) }
```

Comment:
Maybe add a method for this: `emptyNode(node)`. Seems like you use it more than once.